charts/portworx/values.yaml

```diff
@@ -62,8 +62,41 @@ customRegistryURL:
 registrySecret:
 licenseSecret:
 
-monitoring: false
-telemetry: false
+monitoring:
+  prometheus:                         #
+    enabled: false                     # Enable prometheus
+    exportMetrics: true               # Expose the Portworx metrics to an external or operator deployed Prometheus.
+    alertManager: false               # Enable prometheus alert manager
+    resources:                        # Configure Prometheus resource usage such as memory and CPU usage
+      # requests:
+      #   memory: "64Mi"
+      #   cpu: "250m"
+      # limits:
+      #   memory: "128Mi"
+      #   cpu: "500m"
+    replicas: 0                       # Number of prometheus replicas that will be deployed. Default is 1
```

Comment:
As comment mentions Default value is 1, let's change the value.

Reply:
@diptiranjanpx  If the user does not specify a value, it will not be included in the generated `storagecluster` YAML. The operator will then set the default value. I think we should not maintain defaults at two different places.

Reply:
Done Added default replicas to 1